src/awx_plugins/credentials/plugins.py

```diff
@@ -688,10 +688,30 @@
                 'type': 'boolean',
                 'secret': False,
             },
+            {
+                'id': 'ah_path_prefix',
+                'label': gettext_noop('Verify SSL'),
+                'type': 'string',
+                'secret': False,
+                'default': 'galaxy',
+                'help_text': gettext_noop(
+                    'API path used to access the Hub api. Either galaxy, '
+                    'or custom. By default it should be galaxy',
+                ),
+            },
         ],
         'required': ['host'],
     },
     injectors={
+        'extra_vars': {
+            'aap_hostname': '{{host}}',
+            'aap_username': '{{password}}',
+            'aap_password': '{{password}}',
+            'aap_token': '{{oauth_token}}',
+            'aap_request_timeout': '{{request_timeout}}',
+            'aap_validate_certs': '{{verify_ssl}}',
+            'ah_path_prefix': '{{ah_path_prefix}}',
```

Comment:
extra_vars never did anything. That's why they were never put in the injectors before. These appears to correspond to the module arguments, but that implies a syntax like `aap_hostname: "{{ aah_hostname }}"` used in your playbook. This is far from obvious, and I'm not convinced it's useful. It would be very undocumented. Very hard to document.

Reply:
We were writing a blog post on it, but if we just document they are avail, or make them avail, people can use them in places to connect, or use for the lokup plugin or other uses. We shouldn't limite HOW they are used just make them available for the base credential type. I and probably everyone using the validated for a while have also been using custom, just trying to take the standard custom and bring it into the fold.